src/Microsoft.TestPlatform.TestHostProvider/Resources/xlf/Resources.cs.xlf

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../Resources.resx" build-num="1496413687">
+    <body>
+      <trans-unit id="NoTestHostFileExist">
+        <source>No test is available in {0}. Make sure test project has a nuget reference of package "Microsoft.NET.Test.Sdk" and framework version settings are appropriate and try again.</source>
+        <target state="new">Could not find testhost.dll for source '{0}'. Make sure test project has a nuget reference of package "Microsoft.NET.Test.Sdkt". And also check the framework passed when running test.</target>
```

Comment:
No it shouldn't. It has new tag associated with it. The source is correct. Actually in the beginning I had wrong string which I fixed in next commit. It updated the source tag in xlf file but the target tag remained the same. There is no issue with it as it will get updated by translated string.